ranger/config/rifle.conf

```diff
@@ -274,4 +274,5 @@ label pager,  !mime ^text, !ext xml|json|csv|tex|py|pl|rb|js|sh|php  = "$PAGER"
 mime application/x-executable = "$1"
 
 # Move the file to trash using trash-cli.
-has trash-put = trash-put -- "$@"
+label trash, has trash-put = trash-put -- "$@"
+label trash = mkdir -p -- ${"$XDG_DATA_DIR"/ranger-trash:-~/.ranger/trash}; mv -- "$@" ${"$XDG_DATA_DIR"/ranger-trash:-~/.ranger/trash}
```

Comment:
Let's not reinvent the wheel. I'd stick to trash-cli if it's available, nothing otherwise. Any specific reason you want this fallback here?

Reply:
Yeah, I don't install software I don't need. I don't see why an extra option for those who don't have/want trash-cli poses a problem.

Reply:
If they don't need it, they don't need it. If they do need such a functionality, let them install a proper solution. I don't want us to end up with another bulkrename-like solution.

Reply:
How is `mv` a janky solution?